src/pyproxy.c

```diff
@@ -138,6 +138,10 @@ EM_JS(int, pyproxy_new, (int ptrobj), {
     return Module.hiwire_new_value(Module.PyProxies[ptrobj]);
```

Comment:
Is the comment still relevant? Let's move it pyproxy_new below? Also it would be good to update comment docstrings both here and to pyproxy_new  be up to date.

Reply:
You're right, I gonna update both comments.